### DISCUSSION TITLE
Load a previous save of the world

### DISCUSSION THREAD

**Discussion by Absal0n78** [Papela/Nitrox-Cracked-Mod#24] [Q&A]
Hello,

Is it possible to load a certain save of the world? We got big bugs (buildings and vehicles suddenly disappeared + desync issue of one player) and I wonder if loading a previous save would not resolve the issue, or we are f*cked up lol.

**Comment by Papela** [comment]
You can try loading a previous save but I honestly think the corrupted world won't be easy to fix.

**Reply by Absal0n78** [reply]
Understood, then I won't try anything but just start a new one.
Do you have any advise to avoid such bug?

**Reply by Papela** [reply]
Did the bug occur in version 1.8.0.2? or was the world already started in 1.8.0.1?
